test/e2e/scheduling/preemption.go

```diff
@@ -313,6 +317,75 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 		}
 	})
 
+	// 1. Run a low priority pod with finalizer which consumes 1/1 of node resources
+	// 2. Schedule a higher priority pod which also consumes 1/1 of node resources
+	// 3. See if the pod with lower priority is preempted and has the pod disruption condition
+	// 4. Remove the finalizer so that the pod can be deleted by GC
+	ginkgo.It("validates pod disruption condition is added to the preempted pod", func() {
```

Comment:
same note about value: adding job failure policy into the mix would be ideal

Reply:
Possibly, if there is consensus to add it then I prefer to do decouple it from this PR and add before Test Freeze.